test/unit/helpers/dom/AdminConsentRequiredForm.js

```diff
@@ -38,7 +38,7 @@ export default Form.extend({
   scopeNames: function(index) {
     return this.scopeGroupListRow(index).find('.scope-item-text')
       .map(function() {
-        return this.innerText;
+        return this.innerHTML;
```

Comment:
Is `innerText` causing a problem?

Reply:
Fails with innerText due to jsdom, see prev [comment](https://github.com/okta/okta-signin-widget/pull/3418#discussion_r1374788977).